src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1391,5 +1391,6 @@ internal enum ErrorCode
         ERR_TupleExplicitNamesOnAllMembersOrNone = 8204,
 
         ERR_PredefinedTypeMemberNotFoundInAssembly = 8205,
+        ERR_PredefinedTypeNotUniquelyFound = 8206,
```

Comment:
@cston Hold on on further review. From discussion with Aleksey just now, I am making a significant simplification. We'll return null instead and not introduce a new error.